packages/kit/src/views/Discover/Explorer/explorerUtils.ts

```diff
@@ -67,7 +67,7 @@ export const webHandler: WebHandler = (() => {
 })();
 
 export const isValidWebUrl = (url: string) =>
-  /\.(ai|app|art|co|com|club|dev|ee|fi|finance|game|im|info|io|is|it|net|network|news|org|so|xyz)(\/\S*)*$/.test(
+  /^[^/\s]+\.(?:ai|app|art|co|com|club|dev|ee|fi|finance|game|im|info|io|is|it|net|network|news|org|so|xyz)(?:\/[^/\s]*)*$/.test(
```

Comment:
Before:
![WX20230809-231335@2x](https://github.com/OneKeyHQ/app-monorepo/assets/5107241/a0ddf756-f984-482b-998a-81e38511e227)
![WX20230809-231355@2x](https://github.com/OneKeyHQ/app-monorepo/assets/5107241/e5eeb756-dc3c-4a67-9559-81d2983c7e7c)

After:
![WX20230809-231310@2x](https://github.com/OneKeyHQ/app-monorepo/assets/5107241/74399edc-9421-4d7e-990d-e5454432dc38)
![WX20230809-231223@2x](https://github.com/OneKeyHQ/app-monorepo/assets/5107241/9d07c78d-284f-4bb7-bb82-2eff47b65475)

To satisfy linear performance, the meaning of this regexp is modified a bit (matches some extra parts at the beginning of the URL). But it doesn't affect our usage (which tries to validate the URL and auto-correct it).